clouddriver-cloudfoundry/clouddriver-cloudfoundry.gradle

```diff
@@ -28,5 +28,7 @@ dependencies {
   testCompile "org.junit.jupiter:junit-jupiter-engine:${spinnaker.version("jupiter")}"
   testCompile "org.junit.jupiter:junit-jupiter-params:${spinnaker.version("jupiter")}"
 
+  compile 'org.springframework.cloud:spring-cloud-context:1.3.5.RELEASE'
```

Comment:
The versions of `spring-cloud-context` and `spring-cloud-config-server` will have to change, as Cloud versions are closely tied to Boot versions. If we want to hold off on merging this PR until after `boot2` is merged to master, I can update the versions and re-test everything. Otherwise we'll have to update these versions on the `boot2` branch. 

Also see https://github.com/spinnaker/spinnaker-dependencies/commit/4ad3fa68eae1797219bb752043c3ace99f0cd2ac.

Reply:
Sounds good; the `boot2` branch was merged this afternoon